packages/builder/test/fixture-projects/scripts-dir/failing.js

```diff
@@ -1,3 +1,7 @@
+
 const fs = require('fs')
-fs.appendFileSync("output.txt", "failing scripts: script failed\n");
-process.exit(123);
+
+export default async function (runtimeEnv, deployer, accounts) {
+  fs.appendFileSync("output.txt", "failing script: script executed\n");
+  return 123;
+}
```

Comment:
why this script is failing?

Reply:
This script returns a status 123 so I don't use `process.exit(123)` but instead expect the function to return a number.
This will allow the developers to have custom statuses in their CI builds.